test/sim/moves/swallow.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const assert = require("./../../assert");
+const common = require("./../../common");
+
+let battle;
+
+describe("Swallow", function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it("should heal 1/4 health (rounded down) after 1 stockpile", function () {
```

Comment:
The formatting of tests here needs reworked; try to model it after tests from around that area (Sparkling Aria is a good example).